### PR TITLE
Adding a note about inheritance from LAMP plugin.

### PR DESCRIPTION
**Pull request by sk33lz**

### CODE REVIEW

File: docs/plugins/drupal-plugin.md [hunk 1]
```diff
@@ -5,8 +5,9 @@ class: documentation
 permalink: /plugins/drupal-plugin/
 published: true
 ---
-The Drupal plugin provides an easy way for you to configure your Drupal build and to save you some shell scripting. To use the Drupal plugin you must declare `plugin: Drupal` in your `.probo.yaml` file. The plugin's parameters can automate your Drupal build by reverting features, running database updates, clearing caches, or performing other build configuration steps.
+The Probo Drupal plugin provides an easy way to set Drupal site configuration options in a Probo Build to quickly integrate a Drupal website's repository. To use the Probo Drupal plugin you must declare `plugin: Drupal` in your `.probo.yaml` file. The Drupal plugin's parameters can automate some steps specific to Drupal such as reverting features, running database updates, clearing caches, or performing other build configuration steps.
```
Comment (jkaeser) [inline docs/plugins/drupal-plugin.md]: This first sentence has become a run-on. Can you split that into two sentences, please?

File: docs/plugins/drupal-plugin.md [hunk 2]
```diff
@@ -5,8 +5,9 @@ class: documentation
 permalink: /plugins/drupal-plugin/
 published: true
 ---
-The Drupal plugin provides an easy way for you to configure your Drupal build and to save you some shell scripting. To use the Drupal plugin you must declare `plugin: Drupal` in your `.probo.yaml` file. The plugin's parameters can automate your Drupal build by reverting features, running database updates, clearing caches, or performing other build configuration steps.
+The Probo Drupal plugin provides an easy way to set Drupal site configuration options in a Probo Build to quickly integrate a Drupal website's repository. To use the Probo Drupal plugin you must declare `plugin: Drupal` in your `.probo.yaml` file. The Drupal plugin's parameters can automate some steps specific to Drupal such as reverting features, running database updates, clearing caches, or performing other build configuration steps. 
 
+The Probo Drupal plugin inherits all [Probo LAMP plugin](/plugins/lamp-plugin/) configuration options. This allows additional Probo Build steps in your `.probo.yaml` file to layer additional LAMP configuration options and commands on top of the Drupal site specific configuration.
```
Comment (jkaeser) [inline docs/plugins/drupal-plugin.md]: You use "additional" twice in the same sentence, which sounds a little funny.

"This allows ~additional~ **other** Probo Build steps in your `.probo.yaml` file to layer additional LAMP configuration options and commands on top of the Drupal site specific configuration."

**Review by jkaeser — CHANGES_REQUESTED** [review]

**Comment by sk33lz** [comment]
Updated text as per John's comments.

**Comment by sk33lz** [comment]
I added the lamp recipe specific to Drupal to the bottom of the page here, and added an anchor similar to what I just did on the LAMP plugin page.

**Review by jkaeser — APPROVED** [review]

**Pull request merged**